src/views/viewTabs.component.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ViewTabs from './viewTabs.component';
+import { screen } from '@testing-library/react';
+import { renderWithProviders } from '../setupTests';
+import userEvent from '@testing-library/user-event';
+
+describe('View Tabs', () => {
+  const createView = () => {
+    return renderWithProviders(<ViewTabs />);
+  };
+
+  it('lets users switch between tabs', async () => {
+    const user = userEvent.setup();
+    createView();
+
+    // should load Data tab initially
```

Comment:
I couldn't find the information I needed on this one but is this the behaviour we are wanting or is the user expecting to be able to refresh the page and see the same tab they were already on? Is the user's current tab loaded from a session? This would obviously mean we'd need to put this into the store if so.

Reply:
I have not seen a requirement for this and it is not the behaviour of eCAT (which always loads the Data view) so I haven't accounted for it.

@mattjSTFC thoughts?

Reply:
His calendar says A/L. I'm happy to let this get merged since we can't find anything specifically saying this behaviour is incorrect. We can always change it later